modules/rhdh-web-console.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * layered-products-web-console.adc
+
+:_mod-docs-content-type: CONCEPT
+[id="rhdh-web-console_{context}"]
+= {rh-dev-hub} in the {product-title} web console
+
+The {rh-dev-hub} is a platform you can use for building developer portals with helpful templates to simplify the development process. Quick starts are available for you to learn more about how to install and use the developer hub.
```

Comment:
Is there a specific reason that we mentioned templates in the above sentence? Developer Hub also uses plugins to build developer portals and simplify development. I'd suggest changing the sentence to:

The Red Hat Developer Hub is an open platform for building developer portals, containing a supported and opinionated framework. You can use the quick starts to learn more about how to install and use the Developer Hub.

Reply:
I got templates from the first paragraph here https://developers.redhat.com/rhdh if that is not the case we should find out who manages that page and update there too.

Reply:
Link to RHDH docs is below this entry on line 30

Reply:
👍 will address